services/alarm-server/src/main/java/org/phoebus/applications/alarm/server/AlarmServerMain.java

```diff
@@ -135,7 +137,7 @@ private boolean handleShellCommands(final String... args) throws Throwable
             restart.offer(false);
         else if (args.length == 1)
         {
-            if (args[0].startsWith("shut"))
+            if (args[0].equals("shutdown"))
```

Comment:
I would prefer to be lenient with the shell commands.
Both "shutdown" or just "shut", "shutd", ... should be recognized since I've been gotten used to just typing "shut".
If there was another shell command that also started with "shut", then of course we'd have to check more of the text, but the other commands "cd", "ls", ... pose no conflict.

Reply:
OK with the rest

Reply:
> I would prefer to be lenient with the shell commands.
> Both "shutdown" or just "shut", "shutd", ... should be recognized since I've been gotten used to just typing "shut".
> If there was another shell command that also started with "shut", then of course we'd have to check more of the text, but the other commands "cd", "ls", ... pose no conflict.

I agree with Kay here, starts with is a lot more forgiving...

For the other stuff:
I appreciate the logging on the conflict between the settings file and the arguments...it should be very useful for debugging.

Reply:
I thought, both of you made these comments, yes, I've agreed with both of you. Please check the latest changes.  However, I changed `-h` argument in order to keep all options consistently within `Alarm Trio`.  :smiley: